bitrise.yml

```diff
@@ -489,11 +504,36 @@ workflows:
         - test_repetition_mode: retry_on_failure
         - maximum_test_repetitions: "5"
         - scheme: PaymentSheet Example
+        - test_plan: PaymentSheet Example-Shard1
+        - log_formatter: xcbeautify
     - deploy-to-bitrise-io@2: {}
     before_run:
     - prep_all
-    after_run:
-    - notify_ci
```

Comment:
was it intentional to remove this?

Reply:
Yep, we don't need to file JIRAs from this test anymore